src/services/ApiService/Dex/Dex.ts

```diff
@@ -42,154 +53,103 @@ export default class DexService {
   }
 
   public getTokenList = async (): Promise<DexAsset[]> => {
-    const { data: tokenList }: { data: DexAsset[] } = await this.service.get('token-list-full');
+    const {
+      data: { records: tokenList }
+    }: { data: { records: DexAsset[] } } = await this.service.get('swap/v1/tokens');
```

Comment:
What happens when the call fails and `data` is undefined?
A try/catch or return the Promise ?

Reply:
They are wrapped in try catch when used, inside the stateFactory. Do you want to do something other than that?

Reply:
Fine by me